examples/TestClientDefaultEncryption/main.cpp

```diff
@@ -115,33 +117,45 @@ int main()
     // clientConfig->logger = customLogger;
     cout << "Redirecting output stream...done" << endl;
 
-
     // cout << "setCustomConfig..." << endl << endl;
     // client.setCustomConfig(clientConfig);
```

Comment:
remove commented out code, or add now existing param `setCustomConfig` so one does not accidentially comment in method call with default param?